### PR TITLE
(DOCSP-21899): Query > Write

### PR DESCRIPTION
**Pull request by nlarew**
### Jira

- (DOCSP-21899): Query > Write

### Staged Changes (Requires MongoDB Corp SSO)

- [Functions > Query MongoDB > Write](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/fns-mql-write/functions/mongodb/write/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/functions/mongodb/write.txt [hunk 1]
```diff
@@ -13,66 +15,99 @@ Update Documents
...
+which documents to update and :manual:`update operators
+</reference/operator/update>` to describe the changes to apply to
+matching documents.
+
+.. note::
+   
+   While running update operations, {+backend+} temporarily adds a
+   reserved field, ``_id_baas_transaction``, to documents. If you modify
+   data used by your app outside of {+service-short+}, you may need to
+   consider this field. For more information, see :ref:`Transactional
```
Comment (MongoCaleb) [inline source/functions/mongodb/write.txt:79]: Do we want to be more prescriptive here and say "don't touch it, yo!"?

File: source/functions/mongodb/write.txt [hunk 2]
```diff
@@ -271,3 +310,88 @@ elements in an array field based on an array filter:
...
+
+MongoDB supports :manual:`multi-document transactions
+</core/transactions/>` that let you read and write multiple documents
+atomically, even across collections.
+
+To perform a transaction:
+
+1. Obtain and start a client session with ``client.startSession()``.
+
+#. (Optional) Define a read and write settings object for the
```
Comment (MongoCaleb) [inline source/functions/mongodb/write.txt:373]: This one confused me a bit until I looked through the code snippet below. Could you consider something like the following?
> Define options for the transaction, such as readPreference and writeConcern, and then pass this object to the ``session.withTransaciton()`` method described in the next step.

Comment (MongoCaleb) [reply source/functions/mongodb/write.txt:373]: And, actually, do we describe this object anywhere else? What properties are valid, etc.? If so, we need a link to it. If not....we need it. :)

File: source/functions/mongodb/write.txt [hunk 3]
```diff
@@ -271,3 +310,88 @@ elements in an array field based on an array filter:
...
+   transaction. Pass this object to the method called in the next step.
+
+#. Use the ``session.withTransaction()`` method to start a transaction.
```
Comment (MongoCaleb) [inline source/functions/mongodb/write.txt:376]: Consider showing ``session.withTransaction(options)``

**Review by MongoCaleb — COMMENTED** [review]
LGTM but i think there's one place that could use a little bit more clarification.

File: source/functions/mongodb/write.txt (hunk 1 above)
Comment (nlarew) [reply source/functions/mongodb/write.txt:79]: Yes but you actually DO want to touch it! IIRC if you modify a document in a baas_transaction then the field doesn't get cleaned up automatically - you have to make sure to `$unset` it.

**Review by MongoCaleb — APPROVED** [review]
LGTM!

**Pull request merged**